rust/cadeau/src/xmf/vpx/encoder.rs

```diff
@@ -208,12 +204,14 @@ impl VpxEncoderConfigBuilder {
             timebase_den: self.timebase_den.unwrap_or(0),
             threads: self.threads.unwrap_or(0),
         };
-        VpxEncoderConfig(config)
-    }
-}
 
-impl VpxEncoderConfig {
-    pub fn builder() -> VpxEncoderConfigBuilder {
-        VpxEncoderConfigBuilder::new()
+        // SAFETY: The way we build the config ensures that it is always valid.
```

Comment:
you are right, this comment does not make sense. in fact, I believe it is always memory safe to call, for values that is not set, `libvpx` will have sane default. If the configuration is not valid, it will just simply return a null pointer.